xline/src/revision_number.rs

```diff
@@ -17,12 +17,12 @@ impl RevisionNumber {
 
     /// Get the next revision number
     pub(crate) fn next(&self) -> i64 {
-        self.0.fetch_add(1, Ordering::AcqRel).wrapping_add(1)
+        self.0.fetch_add(1, Ordering::Relaxed).wrapping_add(1)
```

Comment:
Why is the ordering not important here? Please comment it above.

Reply:
The `next` method will only be invoked in the `prepare` phase of the command and this phase is executed serially. Therefore, Relaxed ordering is engouh.